pkg/controller/constants/constants.go

```diff
@@ -10,7 +10,7 @@ import (
 
 var (
 	// DefaultImageName is used if a specific image name is not provided
-	DefaultImageName = GetEnvWithDefault("DEFAULT_IMAGE", "infinispan/server:latest")
+	DefaultImageName = GetDefaultInfinispanJavaImage()
 
 	// InitContainerImageName allows a custom initContainer image to be used
 	InitContainerImageName = GetEnvWithDefault("INITCONTAINER_IMAGE", "busybox")
```

Comment:
Is default init container image for os390 has the same name as for x86?

Reply:
@dmvolod good point! This needs a check

Reply:
busybox supports only s390x and ppc64le, I've narrowed down the arch for openJ9

Reply:
@rigazilla do we need to introduce variables for init container in `RELATED_IMAGE_` style or it has the same name?

Reply:
It's a multiarch image, good for all the arch.
I would keep hardcoded for now (the real improvement to this would be to find a proper solution to the read only PV problem)